cql3/statements/describe_statement.cc

```diff
@@ -688,6 +689,33 @@ future<std::vector<std::vector<bytes_opt>>> generic_describe_statement::describe
         co_return serialize_descriptions({index(db, ks_name, _name, _with_internals)});
     }
 
+    auto ks_meta = db.find_keyspace(ks_name).metadata();
```

Comment:
What happens if ks_name doesn't exist? Will it throw an ugly exception? Maybe we should check once that ks_name exists, and if it doesn't, print a nice invalid_request_error saying that? (or maybe we're already going this and I missed it)?

Reply:
I thought it was checked at the beginning but it wasn't. Added a check, so the error is nicer with invalid_request